bin/dev_stop.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+echo "Stopping Argo"
+if [ -e tmp/pids/server.pid ]; then
+    kill -9 $(cat tmp/pids/server.pid)
```

Comment:
i'm still in favor of making this a `kill` instead of a `kill -9`, and printing the actual process ID from `server.pid`.